CHANGELOG.rst

```diff
@@ -0,0 +1,13 @@
+========================================
+Terraform Provider for AAP Release Notes
+========================================
+
+.. contents:: Topics
+
+v1.0.0
+======
+
+Release Summary
+---------------
+
+This is the initial release of the Terraform provider for AAP.
```

Comment:
I was wondering if would make sense to list some of the functionalities the provider supports.